lib/senkyoshi/models/question_bank.rb

```diff
@@ -18,10 +18,6 @@ def canvas_conversion(course, resources)
     end
 
     def setup_question_bank(question_bank, resources)
-      if @items.count.zero?
-        question_bank.description += "Empty Quiz -- No questions
```

Comment:
Interesting that this is in here at all since there is no `description` on a `QuestionBank` object.  Really, I guess this means that this should just be cleanup.